Submitted By: Jim Gifford (jim at linuxfromscratch dot org)
Date: 2004-05-01
Initial Package Version: 4.0.4.1
Origin: Jim Gifford
Upstream Status: Not Sent
Description: Fixes Compile Issue with glibc cvs 2004-04-23
 
--- shadow-4.0.4.1/libmisc/xmalloc.c.orig	2004-05-02 04:53:13.333031035 +0000
+++ shadow-4.0.4.1/libmisc/xmalloc.c	2004-05-02 04:53:22.990258875 +0000
@@ -13,7 +13,6 @@
 RCSID ("$Id: shadow-4.0.4.1-malloc-1.patch,v 1.1 2004/06/19 05:38:15 jim Exp $")
 #include <stdio.h>
 #include "defines.h"
-extern char *malloc ();
 
 char *xmalloc (size_t size)
 {
